Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -172,17 +172,17 @@
       - Biological
     damage:
       groups:
-        Brute: -30 # 10 for each type in the group
+        Brute: -60 # was 10 for each, now 20 for each type in the group, Buffed due to surgery changes (Goobstation)
     bloodlossModifier: -10 # a suture should stop ongoing bleeding
     healingBeginSound:
       path: "/Audio/Items/Medical/brutepack_begin.ogg"
     healingEndSound:
       path: "/Audio/Items/Medical/brutepack_end.ogg"
   - type: Stack
     stackType: MedicatedSuture
-    count: 10
+    count: 15 #Was 10, Buffed due to surgery changes (Goobstation)
   - type: StackPrice
-    price: 20
+    price: 40 #Was 20, Buffed due to limb damage changes (Goobstation)
```

Comment:
Maybe put it to 50? Sutures are rare and hard to get outside of getting logi to buy you 30 of them. It's only fair, given the surgery's ability to remove most "very painful" damage.

Reply:
That would increase the sale value which is the problem we're having with the test fails rn.

Reply:
> That would increase the sale value which is the problem we're having with the test fails rn.

That isn't a massive concern though.

Reply:
Fair, but a crate for 3 advanced kits costs $2k with the updates. I think that should be fine.